keyboards/deemen17/de80/keyboard.json

```
@@ -0,0 +1,997 @@
+{
+    "manufacturer": "Deemen17",
+    "keyboard_name": "DE80",
+    "maintainer": "Deemen17",
+    "bootloader": "uf2boot",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "rows": ["A8", "A10", "A9", "B11", "C14", "C13"],
+        "cols": ["B14", "B13", "B12", "B2", "B1", "B0", "A7", "A6", "A5", "A4", "A3", "A2", "A1", "A0", "A15", "B7", "B6"]
+    },
+    "processor": "STM32F103",
+    "board": "STM32_F103_STM32DUINO",
+    "usb": {
+        "vid": "0xDE17",
+        "pid": "0x80F1",
+        "device_version": "1.0.0"
+    },
+    "indicators": {
+        "caps_lock": "B10",
+        "scroll_lock": "B3"
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 40,
+        "max_brightness": 250,
+        "sleep": true
+    },
+    "ws2812": {
+        "driver": "spi",
+        "pin": "B15"
+    },
+    "community_layouts": [
+        "tkl_ansi_tsangan", 
+        "tkl_ansi_tsangan_split_bs_rshift",
+        "tkl_f13_ansi_tsangan",
+        "tkl_f13_ansi_tsangan_split_bs_rshift"
+    ],
```

add compatible Community Layouts
```suggestion
    "community_layouts": ["tkl_ansi_tsangan", "tkl_ansi_tsangan_split_bs_rshift", "tkl_ansi_wkl", "tkl_ansi_wkl_split_bs_rshift", "tkl_f13_ansi_tsangan", "tkl_f13_ansi_tsangan_split_bs_rshift", "tkl_f13_ansi_wkl", "tkl_f13_ansi_wkl_split_bs_rshift" ],
```